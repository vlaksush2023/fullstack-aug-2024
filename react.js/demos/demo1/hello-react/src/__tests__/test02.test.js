import React from 'react';
import renderer from 'react-test-renderer';
import MyButton from '../TestDemos/MyButton';


test('MyButton works as expected', () => {
   const tree = renderer.create(<MyButton label="Submit Me" />).toJSON()

   expect(tree).toMatchSnapshot();
})