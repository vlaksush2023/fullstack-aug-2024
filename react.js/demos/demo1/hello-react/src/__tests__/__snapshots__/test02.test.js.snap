// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyButton works as expected 1`] = `
<div>
  <button>
    Submit Me
  </button>
</div>
`;
